packages/jsii-pacmak/lib/targets/golang/util.ts

```diff
@@ -1,3 +1,4 @@
+// TODO: Generic utils for naming public/private data types & properties
```

Comment:
I suspect we won't need this function at all -- I've had to remove it almost everywhere it's used. It only seems to apply to package names, and there's a chance we keep pascal case there for readability.